src/uu/chown/src/chown.rs

```diff
@@ -165,17 +165,40 @@ pub fn uu_app() -> App<'static, 'static> {
         )
 }
 
-fn parse_spec(spec: &str) -> UResult<(Option<u32>, Option<u32>)> {
-    let args = spec.split_terminator(':').collect::<Vec<_>>();
+/// Parse the username and groupname
+///
+/// In theory, it should be username:groupname
+/// but ...
+/// it can user.name:groupname
+/// or username.groupname
+///
+/// # Arguments
+///
+/// * `spec` - The input from the user
+/// * `sep` - Should be ':' or '.'
+fn parse_spec(spec: &str, sep: char) -> UResult<(Option<u32>, Option<u32>)> {
+    assert!(['.', ':'].contains(&sep));
+    let args = spec.split_terminator(sep).collect::<Vec<_>>();
     let usr_only = args.len() == 1 && !args[0].is_empty();
     let grp_only = args.len() == 2 && args[0].is_empty();
     let usr_grp = args.len() == 2 && !args[0].is_empty() && !args[1].is_empty();
```

Comment:
I think it's an existing problem, but `chown root:root:root` does nothing because `args.len()` is neither 1 nor 2.

Maybe:
```rust
    let mut args = spec.splitn(2, sep);
    let user = args.next().unwrap_or("");
    let group = args.next().unwrap_or("");
    let uid = if !user.is_empty() {
        ...
    let gid = if !group.is_empty() {
        ...
```